src/caustics/lenses/func/base.py

```diff
@@ -161,3 +165,7 @@ def reduced_from_physical_deflection_angle(ax, ay, d_s, d_ls):
     """
 
     return (d_ls / d_s) * ax, (d_ls / d_s) * ay
+
+
+def td_arcsec2_to_time(d_l, d_s, d_ls, z_l):
```

Comment:
Rename this function to be more clear. td is time_delay I suppose. What is time here? Seconds/hours/days?

Typically, for time delay analsys, time unit is day